src/modelfree/score_agent.py

```diff
@@ -13,6 +19,8 @@
 
 score_ex = Experiment('score')
 
+logging.basicConfig(format='%(levelname)s - %(message)s', level=logging.DEBUG)
```

Comment:
Put this in `main()`.

If I import `score_agent` from another file, I don't want it to override my logging config.

Reply:
Also Sacred already supports some logging config, e.g. you can specify the logging level with the `-l` flag. I'm OK with us overriding this if there's a compelling reason though, but would like to try and centralize logging in one place.

Reply:
I didn't realize Sacred had it's own config (more accurately, I'm still figuring out the nuances of how logging configs waterfall); it works perfectly fine to just remove this line